First part: up through Drankorian

msackton opened the pull request.

msackton committed:
First part: up through Drankorian

tsackton commented on Species/Languages.md:
```diff
@@ -1,27 +1,117 @@
+The common languages of Taelgar spoken by [[Humans|humans]] and the [[Elder Folk]] all derive from the original languages spoken by the first of each species to awaken. For [[Dwarves|dwarves]], [[Elves|elves]], [[Halflings|halflings]], [[Stoneborn|stoneborn]], and [[Lizardfolk|lizardfolk]] this original language has not changed since the dawn of time. For humans, the original tongue has been spli ...
 
-Languages in the world of Taelgar are diverse, and yet the binding influence of magic and the gods have kept them from diverging too far apart. Nonetheless, individual languages described below typically represent clusters or language families, where knowledge of one member is assumed to represent knowledge of all.
+The history of the languages of other sentient creatures is less well known by scholars, but in most cases, whether due to magic, culture, or other factors, linguistic diversity within species is rare. 
```
I think there is no reason not to leave room for goblinoids, at least, to have linguistic diversity, with the hobgoblins in the "main part" of Taelgar mostly speaking the "Cha'mutte's army lingua franca" but leaving the possibility that e.g. hobgoblins in the far east speak a different language. 

But any of the firstborn races or the highly magical races should have a single fixed language, for sure.

tsackton replied:
Maybe just leave this as a background comment or something, and leave the text as written otherwise untouched.

msackton replied:
Yeah, I was thinking "goblinoids" as the reason for "most" not "all". I'll add a background note, and or just write it up as an exception when I get to goblin languages.

tsackton commented on Species/Languages.md:
```diff
@@ -1,27 +1,117 @@
+* **Cymean**: A number of mutually understandable related languages and dialects spoken in the Cymean peninsula. Cymean words are usually based on Spanish or Portugese, but with some Italian. It is usually understood by speakers of Isinguese. 
+* **Isinguese:** The language spoken in the now destroyed lands of the Istabor Alliance, but still spoken in some of the refugee communities including the [[Duchy of Maseau]] and the [[Refounded Alliance of Aurbez]]. Based on Spanish or French, and is usually understood by speakers of Sembaran (especially southern Sembarans) or Cymean. 
+* **Illorian**: The language spoken on the islands of Illoria, it is often understandable to Chardonian speakers, especially those from the [[Apporia|Apporian Peninsula]] or [[Chardon]] itself. No defined real world mapping yet.
+* **~Northern Dialects~**: A set of now-almost extinct languages spoken before the Great War in a number of the northern realms, such [[Volganis]], [[Amani]], the [[Ausberg Empire]], and the [[Republic of Varjatta]]. These languages were usually understood by speakers of Chardonian, and are based on Italian.
```
This is probably not quite right and masks some linguistic complexity here. E.g., Ausberg Empire is much more Germanic-sounding than Italian, and Varjatta and Volganis are almost Slavic-eseque. 

I think the history of the west is not well enough developed yet to be certain here. The Chardonian-like dialects I was thinking about are more say whatever was spoken in the Chasa river valley cities, or on the Apporia peninsula, not the far north beyond Lake Valandros (which was never part of the Drankorian Empire). 

I would leave these unmapped to real world languages, and clarify that the southern languages (south of Lake Valandros) would be mutually intelligble with Chardon but northern languages may be more complex.

tsackton replied:
I think I was thinking that there was potentially some cross-Sentinels cultural connections here, and that e.g. the Ausberg Empire probably had some Zimkovian roots

msackton replied:
Ah, ok. I'll revise to say Chasa River valley specifically here and leave unmentioned the other bits.

msackton replied:
Ah, ok. I'll revise to say Chasa River valley specifically here and leave unmentioned the other bits.

tsackton commented on Species/Languages.md:
```diff
@@ -1,27 +1,117 @@
+
+## Eastros Language Family
+All languages in the Eastros language family derive from ~Unnamed Mariner Language~, the language spoken by the ancient seafaring culture that originally settled the eastern parts of the Green Sea. They use the ~Unnamed Mariner Language~ alphabet.
+
```
I think that Tollish and Sembaran should use the same alphabet, as that is a fairly big barrier. 

My head canon but doesn't have to be real canon is that a lot of the westernmost Eastros groups had an older alphabet replaced by Drankorian, perhaps vaguely paralleling the replacement of Anglo-Saxon runes with the latin alphabet.

So probably Tollish and maybe Vosic (not sure about that name?) now use the Drankorian alphabet, maybe others as well? But probably not Skaegish as they should feel like an old and distinctive language.

msackton replied:
yeah this part I already changed

msackton replied:
yeah this part I already changed - I agree that Tollish is defintely drankorian. Not sure about Vosic, in my head Skaegish and Vosic are more similar than Tollish and Vosic.

tsackton commented on Species/Languages.md:
```diff
@@ -1,27 +1,117 @@
+* **Illorian**: The language spoken on the islands of Illoria, it is often understandable to Chardonian speakers, especially those from the [[Apporia|Apporian Peninsula]] or [[Chardon]] itself. No defined real world mapping yet.
+* **~Northern Dialects~**: A set of now-almost extinct languages spoken before the Great War in a number of the northern realms, such [[Volganis]], [[Amani]], the [[Ausberg Empire]], and the [[Republic of Varjatta]]. These languages were usually understood by speakers of Chardonian, and are based on Italian.
+
+%% Background Note:
```
Will give this a little thought if there is anything missing. But I think pending the history of the west cleanup this is probably right.

tsackton replied:
I've given a bit more thought to western history and I think there is a cultural group missing, the people who never came under Drankorian rule but also weren't Deno'qai. Need to think a bit more about this and the notes of a language doc probably aren't the place so resolving this, but more to come eventually.

msackton committed:
updated rest of human languages

msackton committed:
mostly finished

non-human stuff needs a bit of cleanup, and some thought about what is
canon vs not and also what should go in player-facing vs not

(in particular, might want to keep some stuff secret vs my players)

msackton committed:
a few potential tweaks to goblin.

tsackton commented on Species/Languages.md:
```diff
@@ -1,49 +1,146 @@
+* **Grell**: a corrupted form of Deep Speech
+
+### Extra-Planar Languages
+Time and space do not work on the other planes as they do on the material plane, and each plane tends to be represented by a single language. These are typically difficult, although not impossible, for outsiders to learn.
```
I think these should actually not be too difficult to learn. Many Tyrwinghans might know a little Sylvan, for example, and it would certainly not be unheard of for priests to know or learn Celestial. 

Some of these are probably not really languages in a traditional sense. For sure Celestial doesn't really make sense as a language in Taelgar - I think it is better thought of as a kind of resonance/echo/understanding of the Divine Essence or something. Need to think about this a bit more.
